### PR TITLE
chore: update CHANGELOG to better convey the 2.0.0 issue

### PR DESCRIPTION
**Pull request by conradoplg**
## Motivation

People were confused by the 2.0.0 release disappearance and the RPC authentication change. This hopefully will make it clearer.

### Follow-up Work

Note that this was copied in the [Release notes](https://github.com/ZcashFoundation/zebra/releases/tag/v2.0.1) so any fixes here need to be duplicated there.

### PR Author's Checklist

- [x] The PR name will make sense to users.
- [x] The PR provides a CHANGELOG summary.
- [x] The solution is tested.
- [x] The documentation is up to date.
- [x] The PR has a priority label.

### PR Reviewer's Checklist

- [ ] The PR Author's checklist is complete.
- [ ] The PR resolves the issue.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,39 +8,41 @@ and this project adheres to [Semantic Versioning](https://semver.org).
...
 - The JSON RPC endpoint has cookie-based authentication enabled by default.
-
-### Added
-
-- NU6-related documentation
-  ([#8949](https://github.com/ZcashFoundation/zebra/pull/8949))
-- A cookie-based authentication system for the JSON RPC endpoint
-  ([#8900](https://github.com/ZcashFoundation/zebra/pull/8900),
-  [#8965](https://github.com/ZcashFoundation/zebra/pull/8965))
+  **If you rely on Zebra RPC, wou will need to adjust your config.** The
```
Comment (arya2) [inline CHANGELOG.md]:
```suggestion
  **If you rely on Zebra RPC, you will need to adjust your config.** The
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,39 +8,41 @@ and this project adheres to [Semantic Versioning](https://semver.org).
...
+  simplest change is to disable authentication by adding `enable_cookie_auth =
+  false` to the `[rpc]` section of the Zebra config file; [refer to the
+  docs](https://zebra.zfnd.org/user/lightwalletd.html#json-rpc). (This was added
+  in 2.0.0 but we include here again for clarity)
```
Comment (arya2) [inline CHANGELOG.md]:
```suggestion
  docs for more information](https://zebra.zfnd.org/user/lightwalletd.html#json-rpc) (this was added
  in v2.0.0, but is being mentioned again here for clarity).
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,39 +8,41 @@ and this project adheres to [Semantic Versioning](https://semver.org).
...
 
 Thank you to everyone who contributed to this release, we couldn't make Zebra without you:
 @arya2, @gustavovalverde, @oxarbitrage and @upbqdn.
 
-## [Zebra 2.0.0](https://github.com/ZcashFoundation/zebra/releases/tag/v2.0.0) - 2024-10-25
+## [Zebra 2.0.0](https://github.com/ZcashFoundation/zebra/releases/tag/v2.0.0) - 2024-10-25 - [YANKED]
 
-This release brings full support for NU6.
+This release was intended to support NU6 but was pointing to the wrong version
+of dependencies which would make Zebra panic at NU6 activation. Use 2.0.1 instead.
```
Comment (arya2) [inline CHANGELOG.md]:
```suggestion
of dependencies which would make Zebra panic at NU6 activation. Use v2.0.1 instead.
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -8,39 +8,41 @@ and this project adheres to [Semantic Versioning](https://semver.org).
...
 
 ### Breaking Changes
 
 - Zebra now supports NU6 on Mainnet.
 - The JSON RPC endpoint has a cookie-based authentication enabled by default.
+  **If you rely on Zebra RPC, wou will need to adjust your config.** The
```
Comment (arya2) [inline CHANGELOG.md]:
```suggestion
  **If you rely on Zebra RPC, you will need to adjust your config.** The
```

**Review by oxarbitrage — APPROVED** [review]
Thanks!

**Review by mpguerra — APPROVED** [review]

**Comment by mpguerra** [comment]
@mergify refresh

**Comment by gustavovalverde** [comment]
@mergify refresh

**Comment by mpguerra** [comment]
@mergify refresh

**Comment by gustavovalverde** [comment]
@mergify refresh

**Comment by gustavovalverde** [comment]
@mergify requeue

**Pull request merged**